_tests_/fhirInteractions.ts

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
This file should be moved to a different branch for the follow on work to run against the fixture data.